src/nacl/bindings/crypto_sign.py

```diff
@@ -165,6 +165,31 @@ def crypto_sign_ed25519_sk_to_curve25519(secret_key_bytes):
     return ffi.buffer(curve_secret_key, curve_secret_key_len)[:]
 
 
+def crypto_sign_ed25519_sk_to_pk(secret_key_bytes):
+    """
+    Converts a secret Ed25519 key (encoded as bytes ``secret_key_bytes``) to
```

Comment:
I'd suggest a bit of rewording here, to put the docstring in line with libsodium's documentation at

https://download.libsodium.org/doc/public-key_cryptography/public-key_signatures#extracting-the-seed-and-the-public-key-from-the-secret-key

which correctly talks about "extraction" rather than "conversion"